.travis.yml

```diff
@@ -14,6 +14,11 @@ install: travis_retry composer install --prefer-source
 
 script: composer ci
 
+after_success:
+  - vendor/bin/phpunit --coverage-clover coverage.clover
```

Comment:
Like this its only run when the CI passed, and if I'm not mistaken the build and PR will be marked as success/fail without waiting on the coverage stuff, which takes quite some extra time.

Re one PHP version: sure, go for it. Personally I don't think this is worth my time though I agree it's better